### PR TITLE
Disable Scrims

### PR DESCRIPTION
**Pull request by bwpikaard**

### CODE REVIEW

**Comment by 0c370t** [comment]
Task linked: [CU-29x6x9r Add the ability to toggle scrims](https://app.clickup.com/t/29x6x9r)

**Review by gankoji — APPROVED** [review]

**Review by CodeRedJack — APPROVED** [review]

File: clients/web/src/lib/components/molecules/scrims/AdminScrimTable.svelte [hunk 1]
```diff
@@ -1,38 +1,50 @@
...
-  let targetId: string;
-  let scrimsLocked: boolean = false;
+    let scrimManagementModalVisible = false;
+    let targetId: string;
+    
+    let scrimsAreDisabled: boolean;
+    $: scrimsAreDisabled = $scrimsDisabled.data?.getScrimsDisabled;
 
-  // TODO: Implement Lock Scrim Workflow
+    // TODO: Implement Lock Scrim Workflow
```
Comment (0c370t) [inline clients/web/src/lib/components/molecules/scrims/AdminScrimTable.svelte:23]: Isn't this what this PR is for?

File: clients/web/src/lib/components/molecules/scrims/AdminScrimTable.svelte [hunk 2]
```diff
@@ -46,11 +58,11 @@
         <button
           class="float-right btn btn-outline btn-accent btn-sm"
           on:click={() => {
-            scrimsLocked = !scrimsLocked;
+            toggleScrimsDisabled(!scrimsAreDisabled);
```
Comment (0c370t) [inline clients/web/src/lib/components/molecules/scrims/AdminScrimTable.svelte:61]:
```suggestion
            confirm("Are you sure you want to toggle scrims?") && toggleScrimsDisabled(!scrimsAreDisabled);
```

File: clients/web/src/lib/components/molecules/scrims/AdminScrimTable.svelte (hunk 1 above)
Comment (bwpikaard) [reply clients/web/src/lib/components/molecules/scrims/AdminScrimTable.svelte:23]: Didn't even see that.

**Review by 0c370t — APPROVED** [review]

**Pull request merged**